src/styles/container.scss

```diff
@@ -0,0 +1,36 @@
+.container {
+  max-width: 944px;
+  padding: 50px 40px;
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 46px 48px;
+
+  justify-content: space-between;
+
+  align-items: center;
```

Comment:
The use of justify-content: space-between and align-items: center is more common in flexbox layouts. In a grid layout, these properties might not have the intended effect. Consider whether these properties are necessary or if they should be adjusted for grid-specific alignment.